benchmark/protocol/src/main/scala/Protocol.scala

```diff
@@ -59,7 +59,7 @@ object protocol {
       .codec[MediumW]
       .codec[SmallW]
       .codec[BigW]
-      .specify1[LargeW, LargeW]("identityLarge")
-      .specify1[MediumW, MediumW]("identityMedium")
-      .specify1[BigW, BigW]("identityBig")
+      .specify1("identityLarge", Field.strict[LargeW]("in"), Type[LargeW])
```

Comment:
@ahjohannessen Yes, you are correct, it can be added later. I guess my point was that it might not be worth obsessing too much about how a protocol definition looks like as long as the model is sound like you mentionned.